ChronoVisor/CMakeLists.txt

```diff
@@ -1,24 +1,24 @@
 project(Chronovisor)
 
-add_library(ChronoVisor_lib STATIC
-        include/ChronoVisorServer.h include/ClientRegistryInfo.h
-        include/SerVector.h include/TimeRecord.h
-        src/ChronoVisorServer.cpp src/ClientRegistryRecord.cpp
-        src/TimeRecord.cpp
-        ../ChronoAPI/ChronoLog/src/TimeManager.cpp ../ChronoAPI/ChronoLog/include/TimeManager.h
-        ../ChronoAPI/ChronoLog/src/ClocksourceManager.cpp ../ChronoAPI/ChronoLog/include/ClocksourceManager.h
-        src/ChronicleMetaDirectory.cpp include/ChronicleMetaDirectory.h
-        include/Chronicle.h include/Story.h include/Archive.h include/Event.h
-        ../ChronoAPI/ChronoLog/src/city.cpp ../ChronoAPI/ChronoLog/include/city.h ../ChronoAPI/ChronoLog/include/config.h include/ClientRegistryManager.h src/ClientRegistryManager.cpp)
-target_include_directories(ChronoVisor_lib PUBLIC include
-        ../ChronoAPI/ChronoLog/include
-        ${ROCKSDB_INCLUDE}) #-I/usr/include
-target_link_libraries(ChronoVisor_lib socketpp Serde ${ROCKSDB_LIB} pthread) #_L/usr/bin/mpi.so
-add_dependencies(ChronoVisor_lib socketpp Serde)
+#add_library(ChronoVisor_lib STATIC
+#        include/ChronoVisorServer.h include/ClientRegistryInfo.h
+#        include/SerVector.h include/TimeRecord.h
+#        src/ChronoVisorServer.cpp src/ClientRegistryRecord.cpp
+#        src/TimeRecord.cpp
+#        ../ChronoAPI/ChronoLog/src/TimeManager.cpp ../ChronoAPI/ChronoLog/include/TimeManager.h
+#        ../ChronoAPI/ChronoLog/src/ClocksourceManager.cpp ../ChronoAPI/ChronoLog/include/ClocksourceManager.h
+#        src/ChronicleMetaDirectory.cpp include/ChronicleMetaDirectory.h
+#        include/Chronicle.h include/Story.h include/Archive.h include/Event.h
+#        ../ChronoAPI/ChronoLog/src/city.cpp ../ChronoAPI/ChronoLog/include/city.h ../ChronoAPI/ChronoLog/include/config.h include/ClientRegistryManager.h src/ClientRegistryManager.cpp)
+#target_include_directories(ChronoVisor_lib PUBLIC include
+#        ../ChronoAPI/ChronoLog/include
+#        ${ROCKSDB_INCLUDE}) #-I/usr/include
+#target_link_libraries(ChronoVisor_lib socketpp Serde ${ROCKSDB_LIB} pthread) #_L/usr/bin/mpi.so
+#add_dependencies(ChronoVisor_lib socketpp Serde)
 
-add_executable(ChronoVisor main.cpp include/ClientRegistryManager.h src/ClientInfo.cpp include/ClientInfo.h)
-target_link_libraries(ChronoVisor ChronoVisor_lib socketpp)
-add_dependencies(ChronoVisor ChronoVisor_lib socketpp)
+#add_executable(ChronoVisor main.cpp include/ClientRegistryManager.h src/ClientInfo.cpp include/ClientInfo.h)
+#target_link_libraries(ChronoVisor ChronoVisor_lib socketpp)
+#add_dependencies(ChronoVisor ChronoVisor_lib socketpp)
 
```

Comment:
All these are from the preview implementation based on socketpp. We will remove it completely some time in the future as a standalone task.